.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         node-version: 16.x
 
     - run: yarn install --frozen-lockfile --network-timeout 1000000
+    - run: yarn type-check
```

Comment:
`yarn type-check` is run as part of `yarn build-all` now, that would have made completely `yarn type-check` redundant and it would be confusing to receive type check errors as part of the build step.